docs/Tutorials/Privacy/Configuring-Multi-Tenancy.md

```diff
@@ -35,13 +35,12 @@ IBFT-Network/
 ## 1. Generate a private and public key pair
 
 In the `Node-1` directory [create the `.pem` formatted key pair]. The key pair belongs to the
```

Comment:
This wasn't introduced by this PR but the link isn't working properly here - I can see it at the bottom of the page but it doesn't publish as a link.  Could you fix that as part of this PR.

Reply:
It doesn't like the back ticks used to format ".pem". I resturctured the link, and put the ".pem" as part of the next sentence, outside of the link.